docs/howto/add-barman-server.md

```diff
@@ -0,0 +1,85 @@
+# Add an new Barman node to the exisiting cluster. 
+"Barman
+Allows your company to implement disaster recovery solutions for PostgreSQL databases with high requirements of business continuity.
+
+Taking an online hot backup of PostgreSQL is now as easy as ordering a good espresso coffee."  ~ https://www.pgbarman.org/
+
+## Before you begin.
+Adding a new Barman Server requires following configuration to be done in ansible .
+
+* Add a barman user in the `environments/<env>/vault.yml`
+```
+POSTGRES_USERS:
+    barman:
+      username: 'barman_username'
+      password: 'barman_secret_password'
+```
+
+* Adding a server group (not IP) barman_source group in `environments/<env>/inventory.ini` (It indicates from which servers barman should backup)
```

Comment:
You could probably adjust this to something like:

* Add servers to `pg_barman_source` group in `environments/<env>/inventory.ini`. This indicates which servers barman should backup.

```ini
[pg_barman_source]
host1
host2
```
or
```ini
[pg_barman_source:children]
group1
group2
```